src/runner/api/itemRunner.js

```diff
@@ -360,11 +360,12 @@ var itemRunnerFactory = function itemRunnerFactory(providerName, data, options)
          * This should have the effect to restore the item state.
          *
          * @param {Object} state - the new state
+         * @param {boolean} isInitialStateRestore - state restoring or not
          * @returns {itemRunner}
          *
          * @fires itemRunner#error if the state type doesn't match
          */
-        setState: function(state) {
+        setState: function(state, isInitialStateRestore) {
```

Comment:
I would have considered adding a default value, `isInitialStateRestore = false`, but if you're sure you found all the uses of this function, this comment can be safely ignored.

Reply:
added